src/components/ThirdOjtContent.js

```diff
@@ -0,0 +1,181 @@
+/** @jsxImportSource @emotion/react */
+import { jsx, css } from '@emotion/react'
+import Form from 'react-bootstrap/Form';
+import Stack from 'react-bootstrap/Stack';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { useState } from 'react';
+import '../../src/App.css';
+
+const inputText = css({
+    padding: 7,
+    width: 250,
+    borderWidth: 1.5,
+    borderStyle: 'solid',
+    borderColor: 'black',
+})
+
+const Label = props => {
+    return (
+        <label
+            css={{
+                fontSize: 18,
+                fontWeight: 600,
+                color: 'black',
+                width: 128,
+            }}>
+            {props.labelName}
+        </label>
+    )
+}
+
+const Button = props => {
+    return (
+        <button
+            css={{
+                backgroundColor: '#8a1747',
+                paddingLeft: 50,
+                paddingRight: 50,
+                paddingTop: 15,
+                paddingBottom: 15,
+                borderWidth: 0,
+                borderRadius: 4,
+                fontSize: 18,
+                fontWeight: 600,
+            }}
+            type="submit"
+            disabled={props.isDisabled}
+        >
+            {props.buttonName}
+        </button>
+    )
+}
+
+const schema = Yup.object().shape({
+
+    name: Yup
+        .string()
+        .required("Name is required"),
```

Comment:
msg တွေကို common ထုတ်ရေးကြည့်ပေးပါ 
{attribute} is required လို ကြေငြာထားပီး  ပြန်အစားထိုးတဲ့ပုံစံနဲ့ ရေးပါ